src/backend/distributed/utils/metadata_cache.c

```diff
@@ -160,7 +160,7 @@ static int WorkerNodeCount = 0;
 static bool workerNodeHashValid = false;
 
 /* default value is -1, for coordinator it's 0 and for worker nodes > 0 */
-static int LocalGroupId = -1;
+static int32 LocalGroupId = -1;
```

Comment:
We use `-1` for "not set yet", so I kept this as signed, although in most other places we used unsigned group Id.

Reply:
I'm kind of wondering whether we should standardize on `int32` for group ID, since integers are always signed at the SQL level.

Reply:
done